entities/api/files.py

```diff
@@ -248,17 +247,9 @@ async def validate_file(
     """
     validator = get_validator(file.file_format)
 
-    # Get first 1MB of a file (support plain text or gzip)
-    body = s3_client.get_object(Bucket=file.namespace, Key=file.path, Range="bytes=0-1000000")["Body"]
-    if file.path.endswith(".gz"):
-        with gzip.GzipFile(fileobj=body) as fp:
-            contents = fp.read().decode("utf-8")
-    else:
-        contents = body.read().decode("utf-8")
-
```

Comment:
@morsecodist do you have context on this? The reason for taking the first 1MB is that otherwise, it will try to validate the whole file, which could be in the GBs.

Reply:
Oups, just saw you just moved this logic elsewhere, looks good!